src/components/Chart/Chart.js

```diff
@@ -63,14 +99,34 @@ export const Chart = ({ data: chartInfo }) => {
     return output;
   });
 
-  const newChartInfo = [...chartData, ...seasonLabels, ...episodeLabels];
+  const nChartData = _.map(nData, (data) => {
```

Comment:
we may want to rename this in the future to be more semantic for users looking at our code